rules/dead-code/tests/Rector/Property/RemoveNullPropertyInitializationRector/TypedPropertiesRemoveNullPropertyInitializationRectorTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DeadCode\Tests\Rector\Property\RemoveNullPropertyInitializationRector;
+
+use Iterator;
+use Rector\Core\Testing\PHPUnit\AbstractRectorTestCase;
+use Rector\Core\ValueObject\PhpVersionFeature;
+use Rector\DeadCode\Rector\Property\RemoveNullPropertyInitializationRector;
+
+final class TypedPropertiesRemoveNullPropertyInitializationRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
```

Comment:
To run this test only on PHP 7.4, just add:

```suggestion
     * @requires PHP >= 7.4
     */
```

See [PHPUnit documentation of this feature](https://phpunit.readthedocs.io/en/9.2/incomplete-and-skipped-tests.html#skipping-tests-using-requires).

Reply:
Great job with separated test case!